app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,13 @@
       <header class="panel content-header">
         <%= yield :header %>
         <nav class="content-header__actions-right">
-          <%= link_to "Add Enquiry", new_contact_path, {:remote => true, 'data-toggle' => "modal", 'data-target' => '#modal-window', id: "btnAddEnquiry", class: "button button--dark" } %>
+          <% if request.path_info.include?('/triage') %>
+            <!-- Ask the user to confirm you want to exit triage -->
+            <%= link_to "Add Enquiry", new_contact_path, {:remote => true, 'data-toggle' => "modal", 'data-target' => '#modal-window', id: "btnAddEnquiry", class: "button button--dark" } %>
+          <% else %>
+            <!-- Go straight to Add enquiry page -->
+            <%= link_to "Add Enquiry", new_contact_path, class: "button button--dark", id:"btnAddEnquiry" %>
```

Comment:
This should check if you have contact create permissions now

Reply:
ok good point

Reply:
done